eZ/Publish/Core/Repository/ContentService.php

```diff
@@ -1743,6 +1747,26 @@ protected function copyTranslationsFromPublishedVersion(APIVersionInfo $versionI
         $this->internalUpdateContent($versionInfo, $updateStruct);
     }
 
+    protected function fieldValuesAreEqual(FieldType $fieldType, Value $value1, Value $value2): bool
```

Comment:
Would go for `areFieldValuesEqual` but no strong opinion.

Reply:
> Would go for `areFieldValuesEqual` but no strong opinion.

hmm, tough choice, this is indeed known pattern (though rather with `is...`). I'd keep it as-is now because of readability in this case: `if ( fieldValuesAreEqual() )`